README.md

```diff
@@ -8,8 +8,22 @@ Currently it supports bumping version for a gem.
 ## Bump version
 
 ### Installation
+Dobby require a github app to be installed on your repository or organization wide. If you have already created a github app you can follow instruction from step 3 otherwise follow these steps: 
 
-Add a file to your github workflow `.github/workflows/dobby-action.yml` with following content:
+1. [Create a minimal GitHub App](https://docs.github.com/en/developers/apps/creating-a-github-app), setting the following fields:
+   - Set GitHub App name. 
+   - Set Homepage URL to your github repository.
+   - Uncheck Active under Webhook. You do not need to enter a Webhook URL.
+   - Under Repository permissions: Contents select Access: Read & write.
+   - Under Repository permissions: Pull requests select Access: Read & write. 
+
+2. Create a Private key from the App settings page and store it securely.
+
+3. Install the App on your repository/organization wide.
+
+4. Set secrets on your repository or on organization containing the GitHub App ID, and the private key you created in step 2 as DOBBY_APP_ID, DOBBY_PRIVATE_KEY
```

Comment:
```suggestion
4. Set secrets on your repository or organization containing the GitHub App ID, and the private key you created in step 2 as DOBBY_APP_ID, DOBBY_PRIVATE_KEY
```

Reply:
Is the private key used for both fields? Or is `DOBBY_APP_ID` something else?

Reply:
`DOBBY_APP_ID ` is the github app ID.